Firestore/Source/API/FIRTimestamp.mm

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-#import "Firestore/Source/Core/FSTTimestamp.h"
+#import "FIRTimestamp+Internal.h"
```

Comment:
Only headers in the Public folder should be unqualified. All the rest should be fully qualified.

Reply:
Done.